spaces/daosquare/index.json

```diff
@@ -0,0 +1,11 @@
+{
+  "key": "daosquare",
+  "symbol": "RICE",
+  "name": "DAOSquare",
+  "defaultView": "core",
+  "address": "0xEc0B6AFb3F9a609CEEd67e2CA551A4c573fd45F7",
+  "token": "0xEc0B6AFb3F9a609CEEd67e2CA551A4c573fd45F7",
+  "core": [],
```

Comment:
I recommend you to add some addresses here otherwise change default view to "all"

Reply:
hey , I have slove it and review it plz

Reply:
hey , I have slove it and review it plz @bonustrack